ghost/core/core/server/web/comments/routes.js

```diff
@@ -1,19 +1,22 @@
 const express = require('../../../shared/express');
+const config = require('../../../shared/config');
 const api = require('../../api').endpoints;
 const {http} = require('@tryghost/api-framework');
+const shared = require('../shared');
 
 const bodyParser = require('body-parser');
 const membersService = require('../../../server/services/members');
 
 module.exports = function apiRoutes() {
     const router = express.Router('comment api');
 
+    router.use(shared.middleware.cacheControl('public', {maxAge: config.get('caching:comments:maxAge')}));
```

Comment:
The issue we are solving here is for one specific endpoint and not the whole API. I think we would be most safe to apply this caching rule to the counts endpoint first to unblock the issue and do more research around correct approach for the whole Comments API

Reply:
Added cache just of counts API